src/app/store/remoteTokens.tsx

```diff
@@ -315,23 +316,23 @@ export default function useRemoteTokens() {
     track('fetchTokensFromFileOrDirectory');
     dispatch.uiState.startJob({ name: BackgroundJobs.UI_FETCHTOKENSFROMFILE });
 
-    let remoteData: RemoteResponseData<unknown> | null = null;
     if (files) {
-      remoteData = await readTokensFromFileOrDirectory(files);
+      const remoteData = await readTokensFromFileOrDirectory(files);
       if (remoteData?.status === 'success') {
+        const sortedTokens = applyTokenSetOrder(remoteData.tokens, remoteData.metadata?.tokenSetOrder ?? []);
```

Comment:
```suggestion
        const sortedTokens = applyTokenSetOrder(remoteData.tokens, remoteData.metadata?.tokenSetOrder ?? Object.keys(remoteData.tokens);
```

If no metadata is present we should respect the order of the token sets in the JSON